docs/getting_started.md

```diff
@@ -0,0 +1,99 @@
+# Getting started
+
+## Tools
+
+You'll need the following:
+
+* [node](https://nodejs.org/en/download/)
+
+Install them using your favorite method (`homebrew`, etc).
+
+## Repo
+
+First, [fork the repository](https://github.com/covidatlas/li.git) so you're ready to contribute back.
+
+### 1. Clone, init submodules, and add upstream
+
+Replace `yourusername` below with your Github username:
+
+```
+git clone git@github.com:yourusername/coronadatascraper.git
+cd li
+git remote add upstream git@github.com:covidatlas/li.git
+```
+
+### 2. Install dependencies
+
+```
+npm install
+```
+
+If you get an error message saying you have an incompatible version of
+`node`, you may need to change version.  You can use `n` to change
```

Comment:
nvm is good for this too

Reply:
Thanks, removed the install link, ppl can sort it out. :-)